### ISSUE TITLE
Lucius theme on Mac's default terminal doesn't look right

### ISSUE THREAD

**Issue by Sberm** [Sberm/Transgender.rs#6]
It looks like this on default terminal

<img width="1582" alt="Screenshot 2024-11-08 at 7 39 00 PM" src="https://github.com/user-attachments/assets/3cc7c62f-ef08-409e-ac07-af4fd5a4a38e">

But it's normal when I open up a tmux window on Mac.

<img width="1582" alt="Screenshot 2024-11-08 at 7 41 46 PM" src="https://github.com/user-attachments/assets/72df1352-2752-4c46-9ae6-52cc951cfe81">

I notice that the `$TERM` environment variable is different, on default terminal window:

```bash
Transgender.rs $ echo $TERM
xterm-256color
```

On tmux window:

```bash
Transgender.rs $ echo $TERM
tmux-256color
```

That being said, on Linux machine that displays normally:

```bash
~ $ echo $TERM
xterm-256color
```

So I don't know the reason behind this.

Not tested on windows terminal yet... Can somebody please do that? Oh I forgot nobody cares about this project...

Thanks,
Howard

**Comment by Sberm** [comment]
Switching true color to xterm color works, turns out mac's default terminal doesn't support true color.

**Comment by Sberm** [comment]
This is fixed by commit f93cc1fed06399fbe3a5c5a1fecece031fb1a3ac
